src/objects/GS_Admin_Checklist_Item__mdt.object

```diff
@@ -0,0 +1,224 @@
+    </fields>
+    <fields>
+        <fullName>Link_URL__c</fullName>
+        <description>The URL redirect the link</description>
+        <externalId>false</externalId>
+        <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>The URL redirect the link</inlineHelpText>
+        <label>Link URL</label>
+        <required>false</required>
+        <type>Url</type>
+    </fields>
+    <fields>
+        <fullName>Position__c</fullName>
+        <description>The position of the section item in his section on the Getting Started page</description>
+        <externalId>false</externalId>
+        <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>Is the position of the section item</inlineHelpText>
+        <label>Position</label>
+        <precision>2</precision>
+        <required>true</required>
+        <scale>0</scale>
+        <type>Number</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Primary_Button_Label__c</fullName>
+        <description>The custom label api name used in the Primary Button on a section item on the Getting Started page</description>
+        <externalId>false</externalId>
+        <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>The custom label api name for the Primary Button Label</inlineHelpText>
+        <label>Primary Button Label</label>
+        <length>255</length>
+        <required>false</required>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Primary_Button_Type__c</fullName>
+        <description>The type of action released the Primary Button</description>
+        <externalId>false</externalId>
+        <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>The type of action released the Primary Button</inlineHelpText>
+        <label>Primary Button Type</label>
+        <required>true</required>
+        <type>Picklist</type>
+        <valueSet>
+            <restricted>true</restricted>
+            <valueSetDefinition>
+                <sorted>false</sorted>
+                <value>
+                    <fullName>Disabled</fullName>
+                    <default>true</default>
+                    <label>Disabled</label>
+                </value>
+                <value>
+                    <fullName>sfdc:new-tab</fullName>
+                    <default>false</default>
+                    <label>SFDC New Tab</label>
+                </value>
+                <value>
+                    <fullName>link</fullName>
+                    <default>false</default>
+                    <label>External Page</label>
+                </value>
+            </valueSetDefinition>
+        </valueSet>
+    </fields>
+    <fields>
+        <fullName>Primary_Button_Value__c</fullName>
+        <description>The value used in action released to the Primary Button</description>
```

Comment:
Can you explain more about exactly how this is used? It's a plain text field so there is nothing that defines what type of values are expected and how the value is used on the page.